system/lib/libc/musl/src/stdio/stdout.c

```diff
@@ -1,15 +1,36 @@
 #include "stdio_impl.h"
 
+#if __EMSCRIPTEN__
```

Comment:
Don't we have commend we are supposed to add to all emscripten changes // XXX somthing?

Reply:
Yes, but it's redundant if the code is in an ifdef for `__EMSCRIPTEN__` I think. More readable this way, and all the information is there.

Reply:
OK.. but all our changes should be inside `ifdef __EMSCRIPTEN__` shouldn't they?  Either way lgtm.

Reply:
Yeah, I'm not sure... this change is literally for the emscripten target only, while another change might be a backport or a workaround or something else that is relevant for other targets, and it would be odd to ifdef on emscripten there. But I guess we could do that there too..?

Another way to look at it: I think there's value to keeping the old musl code around in the other ifdef arm, for comparison, but that's not necessarily true in other cases.

Reply:
I know it's a bit late, but changes like this should be listed [in the libc readme](https://github.com/emscripten-core/emscripten/blob/incoming/system/lib/libc/README.md), right?

Reply:
Thanks @VirtualTim, I opened #9419 now.